feat(order tracker) :

Eslam-mahmou opened the pull request:
- add ui order tracker .
- add order tracker package.
- add firebase firestore .

Eslam-mahmou committed:
feat(order tracker) :
- add ui order tracker .
- add order tracker package.
- add firebase firestore .

Feto505 commented on lib/features/orders/presentation/views/track_order_screen.dart:
```diff
@@ -0,0 +1,81 @@
+            InkWell(
+              onTap: () {
+                Navigator.pop(context);
+              },
+              child: Icon(Icons.arrow_back_ios),
+            ),
+
```
use AppIcon is better

Feto505 commented on lib/features/orders/presentation/views/track_order_screen.dart:
```diff
@@ -0,0 +1,81 @@
+                Expanded(child: SizedBox(height: resposiveHeight(8))),
+                ElevatedButton(
+                  onPressed: () {},
+                  child: Text(
+                    "Show Map",
+                    style: AppTextStyles.inter600_18.copyWith(
+                      color: AppColors.whiteColor,
+                    ),
```
hard coded String

Feto505 reviewed (approved):
Outstanding performance......

Eslam-mahmou committed:
feat(order tracker) :
- add ui order tracker .
- add order tracker package.
- add firebase firestore .

Eslam-mahmou committed:
feat(order tracker) :
- add ui order tracker .
- add order tracker package.
- add firebase firestore .

Eslam-mahmou committed:
feat(order tracker) :
- handel stream builder status.
